docs/frontend-shared.md

```diff
@@ -1,67 +1,39 @@
-
-## How it works
-
-Frontend-shared works by creating a symlink via `$ yarn link` between the working dir
-`./frontend-shared/` and the installed dir `node_modules/@hypothesis/frontend-shared/`.
-
 ## Client setup
 
-To set up `frontend-shared` in the client, run
+To set up ``frontend-shared`` in the client, first build the package and then publish to a your local yalc repository. You may need to install yalc first. ``yarn global add yalc ``
 
 ```shell
-$ make dev
+# from the frontend-shared dir
+$ yarn build
+$ yalc publish
 ```
 
-as normal for development in the `client` application.
-
-This will trigger the `frontend-shared` build which:
-- Builds transpiled JSX into the `frontend-shared/lib` directory
-- Creates a linked package in the `node_modules/@hypothesis` folder (this is linked to `frontend-shared/lib`)
-
-When `$ make dev` is running, file changes will automatically trigger a rebuild. To rebuild manually, run:
+In the ``client`` repsitory, link to you local pushed reposotry
 
 ```shell
-$ gulp setup-frontend-shared
+# from the client dir
+$ yalc add  @hypothesis/frontend-shared
 ```
 
-## Usage in other repositories
+This will install the built local version of your ``frontend-shared`` respoitory in the ``client`` without a using symlink.
 
-If you have modified the `frontend-shared` package locally and would like to preview changes in another Hypothesis applications that imports the package--that is, if you want to test uncommitted changes in `frontend-shared` in another local repository before publishing changes--follow these steps:
+_Note_: Currently, there is no support for watchers so any changes made to the ``frontend-shared`` files will need to be rebuilt and re-pushed as follows
 
-1. In this repository's directory
 ```shell
-    $ gulp setup-frontend-shared
+# from the frontend-shared dir
+$ yarn build
+$ yalc push
 ```
 
-1. In the target repository
-```shell
-    $ yarn link @hypothesis/frontend-shared
-```
-
-_Note_: the package will need to be rebuilt after any changes are made to it before those changes can be seen in the consuming repository. This can be done by performing either of the following steps:
-
-  - Run `$ make dev` in this repository, which will **automatically** re-build on any file changes, or
-  - Run `$ gulp build-frontend-shared-js` in this repository to re-build **manually** after changes
-
 #### Removing the link
 
-If you wish to revert back to a published version, as opposed to a local version, you'll have to remove the local link and force the  install:
+If you wish to revert back to the current installed version of ``frontend-shared``, simply run
 
 ```shell
-    $ yarn unlink @hypothesis/frontend-shared
-	$ yarn install --force
+# from the client dir
+$ yalc remove  @hypothesis/frontend-shared
```

Comment:
My normal workflow for unlinking a project is to run `yarn --check-files` which will check the contents of node_modules/ and replace any that don't come from expected sources.